group/edwards25519/fe_test.go

```diff
@@ -0,0 +1,64 @@
+package edwards25519
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.dedis.ch/kyber/v3/xof/blake2xb"
+)
+
+func Test_feToBnZero(t *testing.T) {
```

Comment:
I'd add some additional edge cases (e.g. all 1's) and put those cases in a list, so that it's easy to add new ones (like here: https://github.com/FiloSottile/edwards25519/blob/main/field/fe_test.go#L47).

Reply:
Done !